### ISSUE TITLE
Albums are deleting as soon as they are downloaded

### ISSUE THREAD

**Issue by orcutt989** [Lidarr/Lidarr#4550]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Current Behavior

Albums are deleted as soon as they are downloaded after setting up. Something related to imports and upgrdades. "Upgrades Allowed" is not checked in any of the Quality Profiles. I also tried setting Media Management >> Propers and Repacks to "Do Not Prefer" and that didnt seem to help either.

### Expected Behavior

Albums should not be deleted when they are downloaded.

### Environment

```markdown
- OS: Windows 10
- Lidarr: 2.0.7.3849
- Docker Install: No
- Using Reverse Proxy: No
- Browser: Chrome
- Database: Sqlite 3.36.0
```

### What branch are you running?

Master

### Trace Logs?

[lidarr.txt](https://github.com/Lidarr/Lidarr/files/14175668/lidarr.txt)
[lidarr.trace.txt](https://github.com/Lidarr/Lidarr/files/14175669/lidarr.trace.txt)
[lidarr.debug.txt](https://github.com/Lidarr/Lidarr/files/14175670/lidarr.debug.txt)

### Trace Logs have been provided as applicable. Reports may be closed if the required logs are not provided.

- [X] I have read and followed the steps in the wiki link above and provided the required trace logs - the logs contain `trace` - that are relevant and show this issue.

**Comment by bakerboy448** [comment]
- Duplicate of https://github.com/Lidarr/Lidarr/issues/2746

**Comment by orcutt989** [comment]
Hi @bakerboy448 I see this was closed. I looked at all the issues you linked and none of them seem relevant unfortunately.

I looked through all the issues and it seems like these people are using extra unsupported configs like scripts and rsync after they download files. I am not thinking these are relvant since I'm just downloading and Lidarr is importing and deleting them according to the logs. I am not using any extra processes.
